core-web/libs/dotcms-js/src/lib/core/site.service.ts

```diff
@@ -233,4 +233,5 @@ export interface Site {
     type: string;
     identifier: string;
     archived: boolean;
+    googleMap?: string;
```

Comment:
Why here? is this prop coming back in the site api?

Reply:
Yes, it comes from the site api request. When requesting the actual site data, we have to access the `site.googleMap` attribute. If we do not declare the `googleMap` attribute in the interface, we will get the following error.
TS2339: The property `googleMap` does not exist in the `site` type.

Not all site requests include that attribute, that is why I am assigning the `?`.